CHANGELOG.md

```diff
@@ -2,15 +2,19 @@
 
 ## Version 2.0.1 (Under development)
 
-### App Center
+### AppCenter
 
 * **[Fix]** Handle incorrect usage of `AppCenter.setLogUrl` API to provide readable error message.
 
-### App Center Auth
+### AppCenterAnalytics
+
+* **[Feature]** Support setting latency of sending events via `Analytics.setTransmissionInterval`.
+
+### AppCenterAuth
 
 * **[Fix]** Fix missing proguard rules so that the app does not have to specify them.
 
-### App Center Data
+### AppCenterData
```

Comment:
The change for spaces was intentional and suggested by @achocron on the .net release for mbaas. Next time please do not revert that for next releases.